warehouse/profiles.yml

```diff
@@ -1,11 +1,11 @@
 calitp_warehouse:
-  target: staging
+  target: prod
```

Comment:
oh also this is just the "dbt default" target if you do `dbt run` without anything else

the target is overridden by https://github.com/cal-itp/data-infra/blob/main/airflow/dags/transform_warehouse/dbt_run_and_upload_artifacts.yml#L22 and https://github.com/cal-itp/data-infra/blob/main/warehouse/scripts/run_and_upload.py#L34

Reply:
Ok, I think I fixed this?

Reply:
I still think the default target should be `staging` in case we ever miss passing --target somewhere, or someone accidentally runs it locally with this profiles.yml and no target